packages/graphql-yoga/src/plugins/resultProcessor/regular.ts

```diff
@@ -12,14 +12,18 @@ export function isRegularResult(
   if (!isAsyncIterable(result)) {
     const acceptHeader = request.headers.get('accept')
     if (acceptHeader && !acceptHeader.includes('*/*')) {
-      if (acceptHeader.includes('application/json')) {
-        acceptHeaderByResult.set(result, 'application/json')
+      if (acceptHeader.includes('application/graphql-response+json')) {
```

Comment:
should we also keep the support for `application/json` here?

Reply:
We still support it but we change the order here. If accept header has the newer one with `application/json`, we choose the newer one.